examples/twittermap/web/public/javascripts/map/pointmap.js

```diff
@@ -158,66 +158,95 @@ angular.module('cloudberry.map')
         $scope.scale_x = 0;
         $scope.scale_y = 0;
 
-        //To generate Tweet Popup content from Twitter API (oembed.json?) response JSON
-        function translateOembedTweet(tweetJSON) {
-          var userName = "";
-          try {
-            userName = tweetJSON.author_name;
-          }
-          catch (e){
-            console.log("author_name missing in this Tweet.:" + e.message);
-          }
+        //shows each point's info in Front-end.
+        function translateTweetdatatoShow(tweetJSON) {
```

Comment:
translateTweetDataToShow

Reply:
Sorry for that. Not very precise for Camel-Case rule.